FHIR/sushi
# 1045 sushi does not install dependencies proxy

- Created axiosUtils.ts and created axiosGet function to address posible proxy
- Changed all axios.get calls to axiosGet within /src
- Wrote tests for axiosGet in axiosUtils.test.ts

To test:
- Make sure you have docker installed
- Set up a proxy server through docker: `docker run --name squid -d -p 3128:3128 datadog/squid`
- Set up environment variables `HTTP_PROXY` and `HTTPS_PROXY` as it was done in the issue.
- Use `env|grep HTTP_PROXY` and `env|grep HTTPS_PROXY` to verify that you're using a proxy
- `curl google.com` just to make sure you're connected
- Delete local .fhir cache
- Run sushi on a local sushi project
- If packages download correctly, it is working correctly. If it displays an error regarding the package download failing, then it is breaking.
- `unset HTTP_PROXY` and `unset HTTPS_PROXY` will let you stop using the proxy. Again you can use `env|grep` to verify the two proxies that are not running anymore
- Turn off the docker container: `docker stop squid`
